process/charter.html

```diff
@@ -470,20 +468,20 @@ <h4><a name="join-form">After sending the W3C Decision, Call for Participation</
 <p>After sending the W3C Decision and Call for Participation:</p>
 
 <ul>
-<li>Make sure that IPP has the latest charter link (the W3C Communications Team <a href="https://www.w3.org/admin/dashboard">searches</a> the group and edits information to add a new charter)
+<li>Make sure that Groups DB has the latest charter link (the W3C Communications Team <a href="/admin/dashboard">searches</a> the group and edits information to add a new charter).
```

Comment:
Can it be expanded (to dashboard? database? something else?), or use `abbr`, or link to DB?

Reply:
“Groups DB” is our group’s database, not “dashboard”.

Reply:
Thanks. I updated the suggestion but I'll yield. I take it that those that know / have access to "Groups DB" know what it is. I was just reading it as an outsider and it wasn't clear to me.

Reply:
It’s in the section three that documents for transparency what the team does in the sausage factory.